mesheryctl/internal/cli/root/config/config.go

```diff
@@ -88,3 +96,69 @@ func (v *Version) GetBuild() string {
 func (v *Version) GetCommitSHA() string {
 	return v.CommitSHA
 }
+
+// SetDefaultVersion sets version of config if not specified
+func (ctx *Context) SetDefaultVersion() error {
+	if ctx.Version == "" {
+		// if no version is specified, pick latest imgaes based on channel
+		if ctx.Channel == "edge" {
+			// if channel is edge, pick the edge-latest images
+			ctx.Version = "latest"
+		} else if ctx.Channel == "stable" {
+			// if the channel is stable, pick the version of the latest stable release
+			version, err := GetLatestStableReleaseTag()
+			ctx.Version = version
+			if err != nil {
+				return errors.Wrapf(err, fmt.Sprintf("failed to fetch latest stable release tag"))
+			}
+		} else {
+			return errors.Errorf("unknown channel %s", ctx.Channel)
+		}
+	}
+	return nil
+}
+
+// GetLatestStableReleaseTag fetches and returns the latest release tag from GitHub
+func GetLatestStableReleaseTag() (string, error) {
```

Comment:
This should probably be a part of compose.go and part of the docker compose struct

Reply:
`GetLatestStableReleaseTag` is used to get the latest release tag of meshery, which is used in creating the docker-compose file. This tag is not directly linked with docker-compose. In future, it will also be used for Kubernetes platform to fetch correct manifest files based on version.

Moving only `GetLatestStableReleaseTag` to `helpers.go` will create cyclic dependency.